schemas/README.md

```diff
@@ -0,0 +1,74 @@
+Additionally, once the new struct has been created, another condition will need to be added to the `schemas.go` file to make sure that the correct struct is used when the new topic is specified in the tool.
+
+#### Example of addition of a new topic and Avro schema
+If a new topic called `tx-open` was added with the Avro schema `tx-open.avsc`, with the following structure:
+```
+{
+  \"type\": \"record\",
+  \"name\": \"transaction_open\",
+  \"namespace\": \"namespaceName\",
+  \"fields\": [
+    {
+      \"name\": \"timedate\",
+      \"type\": \"string\"
+    },
+    {
+      \"name\": \"transaction_details\",
+      \"type\": \"string\"
+    }
+  ]
+}
+```
+
+The `Go` struct will look like the following
+
+```go
+type TxClosed struct {
+	Timedate                string  `avro:"timedate"                            json:"timedate"`
+	TransactionDescription  string  `avro:"transaction_details"                 json:"transaction_details"`
+}
+```
+
+A basic rule of thumb to go by is:
+- struct property = "name" of the field within `.avsc` file in camelcase with no underscore to separate words
+- property type  = "type" stated in the `avsc` file
+- avro/json name = will be the same as the name in the `avsc` file
+
+### Changes to an existing topic and Avro schema
+If a change is to be made to an existing topic and its message structure, this means that the Avro schema file (.avsc) will be changed to reflect those changes. A hypothetical example of this could be the `tx-closed` topic. If, there was a decision to add a new field called `description` into the topic message structure, within the [tx-closed](https://github.com/companieshouse/chs-kafka-schemas/blob/master/schemas/tx-closed.avsc) Avro schema file, there will need to be an additional field added to the `fields` array. Whenever this topic is then used in any implementations, the Java Pojo model and/or the Go struct that represents the Avro schema will need to be updated. The same applies to the kafka-management-tool. The `TxClosed` schema will need to be amended to include the `description` field.
+
+#### Example
+An example of the new and updated version of the schema would look like the following
+```
+{
+  \"type\": \"record\",
+  \"name\": \"transaction_closed\",
+  \"namespace\": \"accounts\",
+  \"fields\": [
+    {
+      \"name\": \"attempt\",
+      \"type\": \"int\"
+    },
+    {
+      \"name\": \"transaction_url\",
+      \"type\": \"string\"
+    }
+    {
+      \"name\": \"description\",
+      \"type\": \"string\"
+    }
+  ]
+}
+```
+
+Additionally in the Go structs and/or Java pojos, there would be an extra field added to it to reflect the new field in the Avro schema.
```

Comment:
I think this README would be simpler if it only referenced the changes in this repository. It's a bit confusing after this example to mention "in the Go structs and/or Java pojos" when this repo has no Java POJOs and not have an example of the changes to the TxClosed struct.